docs/examples/simulation.md

```diff
@@ -0,0 +1,392 @@
+If a version is not available, consider using git-hashes.
+
+
+
+# Create Software Configurations
+
+Now that we have our <a href="../../nodes/software" target="_blank">`Software`</a> nodes, we can create <a href="../../subobjects/software_configuration" target="_blank">`SoftwareConfiguration`</a> nodes.
+`SoftwareConfiguration` nodes are designed to let you specify details, about which algorithms from the software package you are using and log parameters for these algorithms. 
+
+The `SoftwareConfigurations` are then used for constructing our <a href="../../nodes/computation" target="_blank">`Computation`</a> node, which describe the actual computation you are performing.
+
+We can also attach <a href="../../subobjects/algorithm" target="_blank">`Algorithm`</a> nodes to a <a href="../../subobjects/software_configuration" target="_blank">`SoftwareConfiguration`</a> node. The <a href="../../subobjects/algorithm" target="_blank">`Algorithm`</a> nodes may contain nested <a href="../../subobjects/parameter" target="_blank">`Parameter`</a> nodes, as shown in the example below.
+
+
+
+```python
+# create some software configuration nodes
+python_config = cript.SoftwareConfiguration(software=python)
+rdkit_config = cript.SoftwareConfiguration(software=rdkit)
+stage_config = cript.SoftwareConfiguration(software=stage)
+
+# create a software configuration node with a child Algorithm node
+openmm_config = cript.SoftwareConfiguration(
+    software=openmm,
+    algorithm=[
+        cript.Algorithm(
+            key="energy_minimization",
+            type="initialization",
+        ),
+    ],
+)
+packmol_config = cript.SoftwareConfiguration(software=packmol)
+```
+
+!!! note "Algorithm keys"
+    The allowed `Algorithm` keys are listed under <a href="https://criptapp.org/keys/algorithm-key/" target="_blank">algorithm keys</a> in the CRIPT controlled vocabulary.
+
+!!! note "Parameter keys"
+    The allowed `Parameter` keys are listed under <a href="https://criptapp.org/keys/parameter-key/" target="_blank">parameter keys</a> in the CRIPT controlled vocabulary.
+
+
+# Create Computations
+
+Now that we've created some <a href="../../subobjects/software_configuration" target="_blank">`SoftwareConfiguration`</a> nodes, we can used them to build full <a href="../../nodes/computation" target="_blank">`Computation`</a> nodes. 
+In some cases, we may also want to add <a href="../../subobjects/condition" target="_blank">`Condition`</a> nodes to our computation, to specify the conditions at which the computation was carried out. An example of this is shown below.
+
+
+```python
+# create a Computation node
+init = cript.Computation(
+    name="Initial snapshot and force-field generation",
+    type="initialization",
+    software_configuration=[
+        python_config,
+        rdkit_config,
+        stage_config,
+        packmol_config,
+        openmm_config,
+    ],
+)
+
+# create a computation node with Condition nodes
+equi = cript.Computation(
```

Comment:
I would spell out the whole variable here to make it crystal clear and no room for mistunderstanding